src/development.cm

```diff
@@ -1,4 +1,4 @@
 Group is
   $/basis.cm
   redprl.cm (bind:(anchor:libs value:../lib/))
-  frontend/frontend.sml
+  frontend/frontend.sml
```

Comment:
I have been wondering how the final new line disappeared in your editor...

Reply:
A mystery...

Reply:
Maybe you could force your editors to add the ending newlines (?).

Reply:
I bet there's a way for me to do that...

Reply:
Update: I got it! :smile: